src/meta/src/hummock/compaction/picker/min_overlap_compaction_picker.rs

```diff
@@ -21,14 +21,14 @@ use risingwave_hummock_sdk::prost_key_range::KeyRangeExt;
 use risingwave_pb::hummock::hummock_version::Levels;
 use risingwave_pb::hummock::{InputLevel, Level, LevelType, SstableInfo};
 
+use super::{CompactionInput, CompactionPicker, LocalPickerStatistic};
 use crate::hummock::compaction::overlap_strategy::OverlapStrategy;
-use crate::hummock::compaction::{CompactionInput, CompactionPicker, LocalPickerStatistic};
 use crate::hummock::level_handler::LevelHandler;
 
 pub struct MinOverlappingPicker {
-    max_select_bytes: u64,
     level: usize,
     target_level: usize,
+    max_select_bytes: u64,
     split_by_table: bool,
```

Comment:
we no need to  `split_by_table` in MinOverlappingPicker. Remove the parameter `split_by_table` and also remove the line 70 condition